lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -205,6 +205,21 @@ for node in $allocated_nodes; do
   ssh "$node" 'for i in /etc/init.d/openstack-* /etc/init.d/openvswitch-switch /etc/init.d/ovs-usurp-config-* /etc/init.d/drbd /etc/init.d/openais; do if test -e $i ; then initscript=`basename $i`; chkconfig -d $initscript; $i stop; fi; done'
 done
 
+
+### Workaround for neutron shortcoming
+# Currently neutron-ns-metadata-proxy is not stoppend on a HA setup. The running process
+# kills the update, so we need to kill it manually. Upstream is working on improving the
+# handling of external processes. Once this is achieved we don't need this part anymore
+get_allocated_nodes neutron-l3
```

Comment:
This won't work: we've disabled the neutron proposal, so this will return nothing ;-)

So either get the list of l3 nodes earlier, or just run this killall bit on all nodes.